airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionUpdaterInput.java

```diff
@@ -21,8 +21,6 @@
   private UUID connectionId;
   @Nullable
   private Long jobId;
-  @Nullable
-  private Integer attemptId;
   private boolean fromFailure;
   private int attemptNumber;
   @Nullable
```

Comment:
The workflow state carries the event Listener and thus is needed for testing

Reply:
Sure ,I also plan to just put the event listener instead of the full state here since we don't need the full state.